pkg/dashboard/store/sql/queries.go

```diff
@@ -42,7 +42,7 @@ FROM users
 WHERE login = $1;
 `,
 		SelectAllUser: `
-SELECT id, login, secret, admin
+SELECT id, login, secret, admin, name, email, access_token, refresh_token, expires, secret, repos, favorite_repos, favorite_services
```

Comment:
Please check the existing other usecases of the SelectAllUser query. Probably it is better to create a new query for this case as returning all these fields might pose problems in existing usecases.